apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/soeknadoversikt/kommerBarnetTilgode/KommerBarnetTilGodeVurdering.tsx

```diff
@@ -1,96 +1,125 @@
 
-// TODO ai: Trekk ut
-export const RedigerWrapper = styled.div`
-  display: inline-flex;
-  cursor: pointer;
-  color: #0056b4;
-  margin-top: 10px;
-
-  .text {
-    margin-left: 0.3em;
+    if (radioError === undefined && begrunnelseError === undefined && svar !== undefined)
+      setKommerBarnetTilGode({ behandlingId, begrunnelse, svar }, (response) => {
+        dispatch(oppdaterKommerBarnetTilgode(response))
+        onSuccess?.()
+      })
   }
 
-  &:hover {
-    text-decoration-line: underline;
+  const reset = (onSuccess?: () => void) => {
+    resetToInitial()
+    setSvar(kommerBarnetTilgode?.svar)
+    setRadioError('')
+    setBegrunnelseError('')
+    setBegrunnelse(kommerBarnetTilgode?.begrunnelse || '')
+    setVurder(kommerBarnetTilgode !== null)
+    onSuccess?.()
   }
-`
 
-const Bold = styled.div`
-  font-weight: bold;
-`
+  return (
+    <VurderingsboksWrapper
+      tittel={''}
+      subtittelKomponent={
+        <>
+          <BodyShort spacing>Boforholdet er avklart og sannsynliggjort at pensjonen kommer barnet til gode?</BodyShort>
+          {kommerBarnetTilgode?.svar && (
+            <Label as={'p'} spacing size="small">
+              {JaNeiRec[kommerBarnetTilgode.svar]}
+            </Label>
+          )}
+        </>
+      }
+      redigerbar={redigerbar}
+      vurdering={
+        kommerBarnetTilgode?.kilde
+          ? {
+              saksbehandler: kommerBarnetTilgode?.kilde.ident,
+              tidspunkt: new Date(kommerBarnetTilgode?.kilde.tidspunkt),
+            }
+          : null
+      }
+      lagreklikk={lagre}
+      avbrytklikk={reset}
+      kommentar={kommerBarnetTilgode?.begrunnelse}
+      defaultRediger={kommerBarnetTilgode === null}
+    >
+      <div>
+        <VurderingsTitle title={'Trenger avklaring'} />
+        <Undertekst gray={false}>
+          Boforholdet er avklart og sannsynliggjort at pensjonen kommer barnet til gode?
+        </Undertekst>
+        <RadioGroupWrapper>
+          <RadioGroup
+            legend=""
+            size="small"
+            className="radioGroup"
+            onChange={(event) => {
+              setSvar(JaNei[event as JaNei])
+              setRadioError(undefined)
+            }}
+            value={svar || ''}
+            error={radioError ? radioError : false}
+          >
+            <div className="flex">
+              <Radio value={JaNei.JA}>Ja</Radio>
+              <Radio value={JaNei.NEI}>Nei</Radio>
+            </div>
+          </RadioGroup>
+        </RadioGroupWrapper>
+        <Textarea
+          style={{ padding: '10px', marginBottom: '10px' }}
+          label="Begrunnelse"
+          hideLabel={false}
+          placeholder="Forklar begrunnelsen"
+          value={begrunnelse}
+          onChange={(e) => {
+            setBegrunnelse(e.target.value)
+            begrunnelse.length > 10 && setBegrunnelseError(undefined)
```

Comment:
se tilsvarende for lengde av begrunnelse over -- jeg tenker uavhenging av om det skal være 10 eller ikke hadde det vært nice å brukt en konstant en plass og ikke inline hver gang vi sjekker opp mot 10 hvis vi skal justere